packages/nhost_flutter_graphql/example/lib/todos_quick_start_example.dart

```diff
@@ -121,6 +121,11 @@ class TodosPage extends StatelessWidget {
       builder: (result, {fetchMore, refetch}) {
         if (result.isLoading) return Text('Loading…');
         if (!result.isConcrete) return SizedBox();
+        if (result.hasException) {
+          return Text('Error encountered while loading todos. Did you setup '
+              'your backend using the quick-start at '
+              'https://docs.nhost.io/quick-start?');
```

Comment:
I get a 404 error when trying to access the url. Did you mean 'https://docs.nhost.io/#quickstart'?